app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/ActionGroup/AdminAssertCategoryGridPageDetailsActionGroup.xml

```diff
@@ -8,16 +8,19 @@
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
               xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
     <actionGroup name="AdminAssertCategoryGridPageDetailsActionGroup">
```

Comment:
Good job! 
I would suggest to use this opportunity and rename the action group to match the best practices:

`Assert{Admin or Storefront}{Functionality}ActionGroup.xml`

So the new name will be `AssertAdminCategoryGridPageDetailsActionGroup`
(see section 11.3.5 in the [tech guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#113-functional-testing))

Thank you!